src/source.h

```diff
@@ -124,9 +124,10 @@ namespace Source {
     
     Gtk::TextIter find_start_of_sentence(Gtk::TextIter iter);
     bool find_open_non_curly_bracket_backward(Gtk::TextIter iter, Gtk::TextIter &found_iter);
+    bool find_close_non_curly_bracket_forward(Gtk::TextIter iter, Gtk::TextIter &found_iter);
     bool find_open_curly_bracket_backward(Gtk::TextIter iter, Gtk::TextIter &found_iter);
     bool find_close_curly_bracket_forward(Gtk::TextIter iter, Gtk::TextIter &found_iter);
-    long open_close_paranthesis_count(Gtk::TextIter iter);
+    long open_close_bracket_count(Gtk::TextIter iter, unsigned int left_bracket, unsigned int right_bracket);
```

Comment:
Well it wasn't an error, at most I would have expected the compiler to throw a warning. As long as it works, it's all good :)